content/blog/2020-03-forcats-0-5-0/index.Rmarkdown

```diff
@@ -0,0 +1,87 @@
+---
+title: forcats 0.5.0
+author: Mara Averick
+date: '2020-03-02'
+slug: forcats-0-5-0
+description: >
+    Announcing the release of forcats 0.5.0 on CRAN. 
+categories:
+  - package
+tags:
+  - tidyverse
+  - forcats
+photo:
+  url: https://unsplash.com/photos/YCPkW_r_6uA
+  author: Jari Hytönen
+---
+
+```{r setup, include = FALSE}
+library(tidyverse)
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We're exceedingly happy to announce the release of [forcats](https://forcats.tidyverse.org/) 0.5.0 on CRAN.
+The goal of the forcats package is to provide a suite of tools that solve common problems with factors, including changing the order of levels or the values.
+
+This release includes improvements to several existing functions, as well as a division of [`fct_lump()`](https://forcats.tidyverse.org/reference/fct_lump.html) into four new functions: `fct_lump_min()`, `fct_lump_prop()`, `fct_lump_n()`, and `fct_lump_lowfreq()`. For a complete inventory of updates in this version, please see the [Change log](https://forcats.tidyverse.org/dev/news/index.html).
+
+You can install forcats with:
+
+```{r install, eval = FALSE}
+install.packages("forcats")
+```
+
+Attach the package by running:
+
+```{r forcats}
+library(forcats)
+```
+
+## New features
+
+### `fct_lump()` function family
+
+Though `fct_lump()` will continue to exist, it has been split into a family of four functions for lumping together levels that meet some criteria as "Other."  
```

Comment:
Lumping seems like a popular activity, and there are many interesting variants. Splitting `fct_lump()` into pieces makes it much easier for this collection to grow over time.